#!/bin/sh /usr/share/dpatch/dpatch-run
## 321-magic-add-powertab.dpatch by Jelmer Vernooij <jelmer@samba.org>
##
## DP: New magic for Power-Tab

@DPATCH@

diff -Naur file-4.20.orig/magic/Magdir/chord file-4.20/magic/Magdir/chord
--- file-4.20.orig/magic/Magdir/chord	2003-03-24 14:48:32.000000000 +0000
+++ file-4.20/magic/Magdir/chord	2007-05-17 16:33:26.000000000 +0000
@@ -7,3 +7,8 @@
 #
 0	string		{title		Chord text file
 
+# Type:	PowerTab file format
+# URL:	http://www.power-tab.net/
+# From:	Jelmer Vernooij <jelmer@samba.org>
+0	string		ptab\003\000	Power-Tab v3 Tablature File
+0	string		ptab\004\000	Power-Tab v4 Tablature File
